packages/apps-config/src/ui/logos/index.ts

```diff
@@ -110,9 +110,9 @@ import moonriver from './nodes/moonriver.svg';
 import nodeMoonrock from './nodes/moonrock.png';
 import moonshadow from './nodes/moonshadow.png';
 import mybank from './nodes/mybank.png';
+import neumann from './nodes/neumann.png';
```

Comment:
icon should be updated too. Use the below new icon.

https://drive.google.com/file/d/1fYML6mTEnQJNiG_R3NGzfv6dQfLLvobM/view?usp=sharing